economics/staking_protocol/main.tex

```diff
@@ -172,7 +170,7 @@ \subsubsection{Sub-stakes}
     \begin{minipage}{0.48\textwidth}
         \centering
         \includegraphics[width=0.99\textwidth]{graphs/two-steps.png}
-        \caption{The stake in figure 3 has been split into two sub-stakes. This was achieved by extending the duration to unlock ($D_2 = t_2 - t$) for a portion (of size $\theta_2$) of the total tokens in the original stake. The two sub-stakes will receive differently sized subsidies.}
+        \caption{The stake in FIG. (3) has been split into two sub-stakes. This was achieved by extending the duration to unlock ($D_2 = t_2 - t$) for a portion (of size $\theta_2$) of the total tokens in the original stake. The two sub-stakes will receive differently sized subsidies.}
```

Comment:
Isn't there some latex syntax for Figures? So if we change the order or insert a new one, we don't need to manually update everything

Reply:
There definitely is, I was spending too much time faffing with latex and wanted to just push the doc. I will fix this in the next update!